src/DoctrineAuditBundle/Reader/AuditReader.php

```diff
@@ -333,6 +365,25 @@ private function filterByTransaction(QueryBuilder $queryBuilder, ?string $transa
         return $queryBuilder;
     }
 
+    private function filterByDate(QueryBuilder $queryBuilder, ?DateTime $startDate, ?DateTime $endDate): QueryBuilder
+    {
+        if (null !== $startDate) {
```

Comment:
It would be even better if you throw an exception when `$endDate` is prior to `$startDate`

Reply:
Added this validation. Thanks